sdk/eventhub/Azure.Messaging.EventHubs/design/api-skeleton-january-2020.md

```diff
@@ -0,0 +1,535 @@
+﻿# .NET Event Hubs Client: Skeleton (January, 2020)
+
+This skeleton represents the public API surface of the .NET Event Hubs client libraries in skeletal form, as they were proposed and reviewed for the preview 6 and January 2020 milestone releases.
+
+## Package: Azure.Messaging.EventHubs
+
+### `Azure.Messaging.EventHubs`
+```csharp
+public class EventHubConnection : IAsyncDisposable 
+{
+    public bool IsClosed { get; protected set; }
+    public string EventHubName { get; }
+    public string FullyQualifiedNamespace { get; }
+    
+    // Basic constructors
+    public EventHubConnection(
+        string connectionString);
+    
+    public EventHubConnection(
+        string connectionString, 
+        EventHubConnectionOptions connectionOptions);
+        
+    public EventHubConnection(
+        string connectionString, 
+        string eventHubName);
+        
+    public EventHubConnection(
+        string connectionString, 
+        string eventHubName, 
+        EventHubConnectionOptions connectionOptions);
+    
+    // Advanced constructor
+    public EventHubConnection(
+        string fullyQualifiedNamespace, 
+        string eventHubName, 
+        TokenCredential credential, 
+        EventHubConnectionOptions connectionOptions = null);
+    
+    // Mocking constructors
+    protected EventHubConnection();
+    
+    public virtual Task CloseAsync(CancellationToken cancellationToken = default);
```

Comment:
Why didn't we mention about `public virtual void Close(CancellationToken cancellationToken = default)` method here?